flags.go

```diff
@@ -291,6 +291,12 @@ func newApp() (app *cli.App) {
 				Usage: "Experimental: Export metrics to the OpenTelemetry collector at this address.",
 			},
 
+			cli.StringFlag{
```

Comment:
we recommend adding new flag only via config file until we make parity between cli and config file. you can refer to this PR on how to add flag to config file: https://github.com/GoogleCloudPlatform/gcsfuse/pull/1800/files

Reply:
Done.